services/docker-compose.development.yml

```diff
@@ -47,5 +47,10 @@ services:
       - '121_service_node_modules:/home/node/app/node_modules'
       - './121-service:/home/node/app'
 
+  121-redis:
+    image: 'redis:alpine'
```

Comment:
No version-number?

We can only use the one that is supported by the Azure service we'd use, right?